app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
```

Comment:
It's more common to put python standard modules above local imports. For convenience you can use isort linter or configure flake to check imports.